.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -7,7 +7,7 @@ on:
     types: [checks_requested]
 
 concurrency:
-  group: ${{ github.head_ref || github.ref }}
+  group: ${{ github.event.pull_request.head.ref }}
```

Comment:
Can anyone think of a use for `github.ref`?

Reply:
I _think_ what you have here is correct and `github.ref` isn't needed 🤞

Reply:
Eh? https://github.com/CrowdStrike/glide-core/pull/857/commits/f04ceff9efac22b17ee42c5b2b1f990f34a51056

Reply:
Should be good, I think.

Reply:
https://github.com/CrowdStrike/glide-core/actions/runs/14625747971/workflow

And now we know. Commit incoming.

Reply:
And [now](https://github.com/CrowdStrike/glide-core/actions/runs/14625968397/workflow) we know.